README.md

```diff
@@ -144,55 +147,60 @@ docker run \
 -e TARGET_PROJECT_KEY=xxxx \
 -e TARGET_CLIENT_ID=xxxx \
 -e TARGET_CLIENT_SECRET=xxxx \
-commercetools/commercetools-project-sync:3.2.1 -s all
+commercetools/commercetools-project-sync:3.2.2 -s all
 ```
   
 
 ### Examples   
  - To run the all sync modules from a source project to a target project
    ```bash
-   docker run commercetools/commercetools-project-sync:3.2.1 -s all
+   docker run commercetools/commercetools-project-sync:3.2.2 -s all
    ```
    This will run the following sync modules in the given order:
- 1. `Type` Sync and `ProductType` Sync in parallel.
+ 1. `Type` Sync and `ProductType` Sync and `States` in parallel.
```

Comment:
Why we have choose to run this in parallel?

Reply:
Things are faster when running in parallel, so that's why running in parallel.